src/machine/machine_atsamd51.go

```diff
@@ -1806,6 +1809,14 @@ func (usbcdc *USBCDC) WriteByte(c byte) error {
 
 			if ok {
 				break
+			} else if usbcdcTxTimeout < UART0.waitTxcTimer {
+				mask := interrupt.Disable()
+				UART0.waitTxc = false
+				UART0.waitTxcTimer = 0
+				usbcdc.TxIdx.Set(0)
+				usbLineInfo.lineState = 0
```

Comment:
I don't know if I can initialize `usbLineInfo.lineState` .
However, in my environment, there was no problem because cdcSetup() was called when I connected again.

env: windows10 + teraterm 4.105

Reply:
> there was no problem because cdcSetup() was called when I connected again

That is what I would expect to happen when it is working correctly. I will try this branch on Linux/macOS.

Reply:
Seems to work as expected on both.